config.sample.yaml

```diff
@@ -46,6 +46,18 @@ log:
 #  # jaeger collector address to send application traces
 #  jaeger_addr: "http://localhost:14268/api/traces"
 
+scheduler:
+  versions:
+    supported:
+      # 1 DAG template can be used by multiple airflow versions,
+      # this config contains mapping from the dag template version to list
+      # of all supported Airflow minor version for that template.
+      dag:
+      - "2.1": ["2.1", "2.2", "2.3"]
```

Comment:
mapping the versioning like this will help us to minimize template duplication, but it needs us to manage it in the future. I'm just thinking, instead of defining the mapping as a config, can we put the mapping in `template.go`? as I believe the mapping is tightly coupled with our code implementation and it's not related to the configuration. wdyt?

Reply:
I'm also a bit unsure if this can be stored in config, since the template choice logic is tailored in the template.go